src/shared/components/deletion-confirmation/DeletionConfirmation.scss

```diff
@@ -3,15 +3,15 @@
 .deleteConfirmationContainer {
   display: flex;
   align-items: center;
-  justify-content: var(--container-justify-content, flex-end);
+  justify-content: flex-end;
```

Comment:
I still don't get it — what if you removed this CSS rule at all?

If the problem is that the component behaves like this:

![image](https://user-images.githubusercontent.com/6834224/190583806-e2fcf135-db3a-4277-88c3-ea7b4d6d3351.png)

and you are trying to make it to behave like this:

![image](https://user-images.githubusercontent.com/6834224/190583882-59e1ce44-04b1-4087-b98a-8f05655e6229.png)

then why not make it behave like this instead:

![image](https://user-images.githubusercontent.com/6834224/190583976-ef0bfce9-7803-49f5-83b7-da48fb7d3113.png)

by changing the `display` property from `flex` to `inline-flex`?

Reply:
Agreed to leave some of the alignment/display to the parent but also make it as an optional props whether to align the content right or not